services/core-api/app/api/now_applications/resources/administrative_amendment_list_resource.py

```diff
@@ -107,21 +108,32 @@ def post(self):
 
             db.session.add(new_app)
 
-            # copy contacts
-            if permit.permittee_appointments:
-                application_appt = []
-                for mine_appt in [
-                        party for party in permit.permittee_appointments
-                        if not party.end_date or party.end_date > datetime.now(timezone.utc).date()
-                ]:
-                    new_app_appt = NOWPartyAppointment(
-                        mine_party_appt_type_code=mine_appt.mine_party_appt_type_code,
-                        now_application_id=new_app.now_application_id,
-                        party_guid=mine_appt.party_guid)
+            # contacts
+            application_appt = []
+            # if there is a prior imported NoW, copy contacts from it.
+            if application and application.contacts:
+                now_appts = [
+                        party for party in application.contacts
+                        if not party.end_date or party.end_date > datetime.now(timezone.utc).date() # valid?
```

Comment:
I think we do not need this comment anymore
"# valid?"

Reply:
I still have the question, if we are copying contacts from the prior NoW, does it matter what end date they had set?

Alternatively, was the intent to copy all current contacts from the mine? **Not** the prior NoW's contacts?